examples/tools/bevymark.rs

```diff
@@ -209,19 +222,19 @@ fn spawn_birds(
 }
 
 fn movement_system(time: Res<Time>, mut bird_query: Query<(&mut Bird, &mut Transform)>) {
-    for (mut bird, mut transform) in bird_query.iter_mut() {
+    bird_query.for_each_mut(|(mut bird, mut transform)| {
```

Comment:
Since this is `bevymark`, I'm reasonably happy using `for_each` here. 
Other examples obviously shouldn't use it, since it's less idiomatic.